app/pkg/email/mailgun/mailgun.go

```diff
@@ -33,6 +39,30 @@ func (s *Sender) Send(ctx email.Context, templateName string, params email.Param
 	return s.BatchSend(ctx, templateName, params, from, []email.Recipient{to})
 }
 
+// Try getting the base URL of the Mailgun API using Environment vars and the Sender's domain
+// Fall back to the URL for the US region if that fails to maintain compatibility with older installs
+func (s *Sender) GetBaseURL() string {
+	var regionCode = env.Config.Email.Mailgun.Region
+	regionCode = strings.ToUpper(regionCode)
+
+	// Default to the US domain if no region code was provided (ENV not set)
+	// or if the provided code isn't valid
+	if len(regionCode) < 1 {
+		regionCode = "US"
+	} else if len(baseURLs[regionCode]) < 1 {
+		s.logger.Warnf(
+			"Unknown Mailgun region code '@{Code}' configured - falling back to 'US'", 
+			log.Props{
+				"Code": env.Config.Email.Mailgun.Region,
```

Comment:
you can reuse regionCode variable here :)
```suggestion
				"Code": regionCode,
```

Reply:
Ah, I think I should have commented that line since it's not that obvious.
If I were to set the region code to something like "Usa" the error message would tell me that the code 'USA' is unknown which might appear strange since I used a mixed case value.
I decided to not reuse regionCode at this point to prevent any confusion.

From a functional standpoint it's mostly if not completely cosmetic though, if you still think it's better to change it I can do that, np.

Reply:
That's a good point, haven't thought about that. Let's keep it as is :)